files/manifests/k8s-api-server.yaml

```diff
@@ -53,6 +53,7 @@ spec:
     - --logtostderr=true
     - --tls-cert-file=/etc/kubernetes/ssl/apiserver-crt.pem
     - --tls-private-key-file=/etc/kubernetes/ssl/apiserver-key.pem
+    - --tls-cipher-suites=TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305,TLS_RSA_WITH_AES_256_GCM_SHA384,TLS_RSA_WITH_AES_128_GCM_SHA256
```

Comment:
This needs to be tested - I am uncertain if the apiserver will be happy with this way of passing them in.

Reply:
I believe you are correct, updated to use the kubelet config files. Still have to try it out

Reply:
Actually, it looks like this method is deprecated for the kubelet but not for apiserver. So I think I need both
https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/